core/src/test/java/org/apache/accumulo/core/conf/ConfigSanityCheckTest.java

```diff
@@ -93,4 +91,18 @@ public void testFail_cipherSuiteNotSetKeyAlgorithmSet() {
     m.put(Property.CRYPTO_CIPHER_KEY_ALGORITHM_NAME.getKey(), "AES");
     ConfigSanityCheck.validate(m.entrySet());
   }
+
```

Comment:
It would be nice to add the combos that are expected to pass to the `testPass()` function.  For example when CRYPTO_MODULE_CLASS and CRYPTO_SECRET_KEY_ENCRYPTION_STRATEGY_CLASS are both null AND when both are not null.

Reply:
Agreed.